SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/DeploymentLauncher/DeploymentLauncher.cs

```diff
@@ -171,47 +194,50 @@ private static int CreateDeployment(string[] args, bool useChinaPlatform)
                     return 0;
                 }
 
-                if (DeploymentExists(deploymentServiceClient, projectName, simDeploymentName))
-                {
-                    StopDeploymentByName(deploymentServiceClient, projectName, simDeploymentName);
-                }
-
-                // we are using the main deployment snapshot also for the sim player deployment, because we only need to specify a snapshot
+                // we are using the main deployment snapshot also for the sim player deployment(s), because we only need to specify a snapshot
```

Comment:
```suggestion
                // We are using the main deployment snapshot also for the sim player deployment(s), because we only need to specify a snapshot
```